zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DbReader.scala

```diff
@@ -49,7 +49,7 @@ object DbReader {
          	NODEPOOL AS np INNER JOIN KUBERNETES_CLUSTER AS cluster
          	on cluster.id = np.clusterId
          	where np.status != "DELETED" and np.status != "ERROR"
-         	""".query[NodepoolToScan]
+         	""".query[Option[NodepoolToScan]]
```

Comment:
Should the sql in `activeNodepoolsQuery` and `activeK8sClustersQuery` be updated to exclude azure?

Reply:
for zombie monitor, I made [this](https://github.com/broadinstitute/leonardo-cron-jobs/pull/235/files#diff-8a5ea2275ca26bd7ff749af0903692597a4896d0857ebf98e4f66ba7fe553e2bR172-R191) change...so it's kinda equivelant